.github/workflows/go.yml

```diff
@@ -7,19 +7,18 @@ on:
     branches: [ master ]
 
 jobs:
-
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
+      - uses: actions/checkout@v3
 
-    - name: Set up Go
-      uses: actions/setup-go@v3
-      with:
-        go-version: 1.18
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: '1.17'
 
-    - name: Build
-      run: go build -v ./...
+      - name: Build
+        run: go build -v ./...
 
-    - name: Test
-      run: go test -v ./...
+      - name: Test
+        run: go test -v ./...
```

Comment:
I have not been through all of the code but any reason why this is not `go run github.com/onsi/ginkgo/v2/ginkgo -r -randomize-all`, especially since you have it declared in tools.

Reply:
No reason. The workflow file was the default, provided by github.
Changed.